docs/authorization-introduction.md

```diff
@@ -0,0 +1,124 @@
+# Authorization Introduction
+
+This document provides a general overview on what are the terms used in the Udaru authorization system and describes the structure and functionality of the authorization application.
+
+The Udaru authorization system has two main functional components:
+- Using the PBAC authorization mechanism for internal use to authorize the system internal parts against themselves,
+- Provide PBAC authorization service for external applications.
+
+## Authorization terms
+
+The access control management evolved from the need to mitigate the risk of unauthorized access to data.
+
+Several control models were built during time to approach the various access control needs: Access Control Lists, Role Based Access Control, Attribute Based Access Control, Policy Based Access Control, Risk Adaptive Access Control. A summary of these models is made on [Survey on access control models][], the document offers a description of the PBAC model that Udaru uses.
+
+Udaru is based on the [node-pbac][] module, module that as the developer claims is inspired and mostly compatible with the [AWS IAM][] policy engine.
```

Comment:
`[...] module, module that as the developer claims [...]`

I would rephrase it like:

`[...] module. It is inspired by and mostly compatible with the [AWS IAM][] policy engine.`